src/components/Footer/FooterExtendedNavList/FooterExtendedNavList.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react'
+import classnames from 'classnames'
+
+export type ExtendedNavLinksType = React.ReactNode[][]
+
+type FooterExtendedNavListProps = {
+  /* 
+    Multidimensional array of grouped nav links. Sub-arrays are column sections, first element is used as a heading.
+  */
+  nestedLinks: ExtendedNavLinksType
+}
+
+export const FooterExtendedNavList = ({
+  className,
+  nestedLinks,
+}: FooterExtendedNavListProps &
+  React.HTMLAttributes<HTMLElement>): React.ReactElement => {
+  const classes = classnames('grid-row grid-gap-4', className)
+
+  const [isMobile, setIsMobile] = React.useState(false)
+  const [sectionsOpenState, setSectionsOpenState] = useState<boolean[]>(
+    Array(nestedLinks.length).fill(false)
+  )
+
+  useEffect(() => {
+    const isClient = window && typeof window === 'object'
```

Comment:
🤷‍♀️  I tried this after first implementing with an `isMobile` prop. It seemed strange, once I built out stories, to have a component that didn't expand itself on a wider screen. 

For reference mobile styles only start happening via CSS `.usa-footer--big .usa-footer__primary-content--collapsible.hidden .usa-list--unstyled {
    display: none;
}`

Happy to go back to to `isMobile` being a prop, would just remove the hook to implement and adjust tests. Just wanted to try out another option. Thoughts on this?

Reply:
I think the only advantage to having isMobile being a prop, is it allows whoever is implementing the component to have more control over the component. I think this is in general what we have been attempting to do, but 🤷‍♀️

Reply:
I'm just gonna make it a prop for simplicity at this point and to move this along.

Reply:
Yeah, this is always a tricky problem. It looks like you've pretty closely replicated what USWDS is doing with their JS, so I say this is good for now. perhaps a future enhancement could be adding an `isMobile` prop that overwrites this behavior, for users who want more control?